### DISCUSSION TITLE
catalog .properties for worker and coordinator?

### DISCUSSION THREAD

**Discussion by pithecuse527** [trinodb/trino#17745] [Q&A] [answered]
Hello, this is my first time to study and use Trino.

I'm uncertain about the appropriate placement of the .properties file for the catalog, whether it should be located in the worker or the coordinator node. 

Consider the following scenario: if I want to designate SERVER1 as the coordinator and SERVER2, SERVER3, SERVER4 as workers, and I intend to add a new catalog to the Trino cluster, where should I place the specified .properties file? Is it necessary to include it on every server, regardless of its role as a coordinator or worker? 

Moreover, what will happen if I skip to include the catalog .properties file on certain workers or the coordinator?

Thank you in advance 🙇‍♂️

**Comment by hashhar** [comment, marked as answer]
The catalogs need to be present on all nodes - you might be able to get things to work with catalogs on coordinator + at-least 1 worker but it's harder to manage the system that way even if it works.
